editor/gui/editor_bottom_panel.cpp

```diff
@@ -246,9 +270,29 @@ EditorBottomPanel::EditorBottomPanel() {
 	bottom_hbox->set_custom_minimum_size(Size2(0, 24 * EDSCALE)); // Adjust for the height of the "Expand Bottom Dock" icon.
 	item_vbox->add_child(bottom_hbox);
 
+	left_button = memnew(Button);
+	left_button->set_tooltip_text(TTR("Scroll Left\nHold Shift to scroll to the begin.\nHold Ctrl to scroll one page."));
```

Comment:
You can make a `const String` to store this message, so it's not repeated.

Reply:
`Scroll Left/Right` and `to the begin/end` are distinct. Did you mean to suggest using vformat() like other controls?